e2e-tests/cypress/e2e/journeys/insurance/account/create/your-details/account-create-your-details-validation.spec.js

```diff
@@ -97,7 +98,7 @@ context('Insurance - Account - Create - Your details page - form validation', ()
       yourDetailsPage[LAST_NAME].input().should('have.focus');
 
       partials.errorSummaryListItemLinks().eq(2).click();
-      yourDetailsPage[EMAIL].input().should('have.focus');
+      accountFormFields[EMAIL].input().should('have.focus');
```

Comment:
can we use my cy validation command here?

Reply:
Great shout - updated